app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/VaultAddEditLoginItems.kt

```diff
@@ -518,13 +520,30 @@ private fun PasswordRow(
 private fun PasskeyField(
     creationDateTime: Text,
     modifier: Modifier = Modifier,
```

Comment:
The modifier, should always be the first property with a default value after the properties without default values.